src/main/java/com/minecolonies/coremod/client/gui/townhall/WindowTownHallColonyManage.java

```diff
@@ -149,7 +151,13 @@ public void onCreate()
         new VanillaParticleMessage(pos.getX(), pos.getY(), pos.getZ(), ParticleTypes.DRAGON_BREATH).onExecute(null, false);
         Minecraft.getInstance().level.playSound(Minecraft.getInstance().player, new BlockPos(Minecraft.getInstance().player.position()),
           SoundEvents.CAMPFIRE_CRACKLE, SoundSource.AMBIENT, 2.5f, 0.8f);
-        Network.getNetwork().sendToServer(new CreateColonyMessage(pos));
+        boolean claim  = false;
```

Comment:
different name than claim here, it sounds like chunk claiming but its about restoring a building, maybe abandoned?